src/ImageSharp/Formats/Png/PngChunk.cs

```diff
@@ -35,7 +35,7 @@ public PngChunk(int length, PngChunkType type, IManagedByteBuffer data = null)
         /// Gets the data bytes appropriate to the chunk type, if any.
         /// This field can be of zero length or null.
         /// </summary>
-        public IManagedByteBuffer Data { get; }
+        public IMemoryOwner<byte> Data { get; }
```

Comment:
Nit: ideally `PngChunk` should implement `IDisposable` to clarify ownership rules. It can keep `IMemoryOwner<byte>` private then, and expose a `Span` property.

Reply:
I'll have a look at that.